firmware/src/app.c

```diff
@@ -80,14 +81,26 @@ void APP_Initialize(void) {
   // TODO(simon): remove this.
   Logger_Initialize(NULL, PAYLOAD_SIZE);
   Logger_SetState(true);
+
+  SPIRGBConfiguration spi_config;
+  spi_config.module_id = SPI_ID_1;
+  spi_config.baud_rate = 1000000;
+  spi_config.use_enhanced_buffering = true;
+  SPIRGB_Init(&spi_config);
+
+  // Send a frame with all pixels set to 0.
```

Comment:
Yeah, the SPI bus seems to spew random data on boot, so you get multi-colored rainbows. Resetting to black seems the most sensible thing to do.